cmd/kar-controllers/app/options/options.go

```diff
@@ -48,6 +52,10 @@ func (s *ServerOption) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&s.Kubeconfig, "kubeconfig", s.Kubeconfig, "Path to kubeconfig file with authorization and master location information.")
 	fs.BoolVar(&s.Dispatcher,"dispatcher",s.Dispatcher,"set dispather mode(true) or agent mode(false)")
 	fs.StringVar(&s.AgentConfigs, "agentconfigs", s.AgentConfigs, "Paths to agent config file:deploymentName separted by commas(,)")
+	fs.BoolVar(&s.DynamicPriority,"dynamicpriority", s.DynamicPriority,"--dynamicpriority=false/true")
+	fs.BoolVar(&s.Preemption,"preemption", s.Preemption,"--preemption=false/true")
+	fs.BoolVar(&s.SendUpdate,"sendupdate", s.SendUpdate,"--sendupdate=false/true")
+	fs.BoolVar(&s.Demo,"demo", s.Demo,"--demo=false/true")
```

Comment:
Please add description text to new vars.  Reference lines above for examples.  Thanks!

Reply:
Added more detailed description test.

Reply:
Looks good.  I added some addition suggestions in a different comment.  Closing this comment and differing to new comment.